### ISSUE TITLE
JMSServer/PersistentStore is not set as per the model during domain creation.

### ISSUE THREAD

**Issue by brahankv** [oracle/weblogic-deploy-tooling#869]
With WDT release 1.9.11, the persistent store is discovered properly,
```
        JMSServer:
            BPMJMSServer:
                MessageBufferSize: 100000
                PersistentStore: BPMJMSFileStore
                Target: soa_server1
            JRFWSAsyncJmsServer:
                MessagesMaximum: 200000
                PersistentStore: JRFWSAsyncFileStore
            SOAJMSServer_auto_1:
                PersistentStore: SOAJMSJDBCStore_auto_1
                Target: soa_server1 (migratable)
            SOAJMSServer_auto_2:
                PersistentStore: SOAJMSJDBCStore_auto_2
                Target: soa_server2 (migratable)
            SOAJMSServer_auto_3:
                PersistentStore: SOAJMSJDBCStore_auto_3
                Target: soa_server3 (migratable)
            UMSJMSServer_auto_1:
                PersistentStore: UMSJMSJDBCStore_auto_1
                Target: soa_server1 (migratable)
            UMSJMSServer_auto_2:
                PersistentStore: UMSJMSJDBCStore_auto_2
                Target: soa_server2 (migratable)
            UMSJMSServer_auto_3:
                PersistentStore: UMSJMSJDBCStore_auto_3
                Target: soa_server3 (migratable)
```
But after creation of domain with the discovered model, the config.xml the persistent store is not set as per the model. Seems it is set to default file store.
```
      <jms-server>
        <name>BPMJMSServer_auto_1</name>
        <target>soa_server1 (migratable)</target>
        <persistent-store>BPMJMSFileStore_auto_1</persistent-store>
        <message-buffer-size>100000</message-buffer-size>
      </jms-server>
      <jms-server>
        <name>UMSJMSServer_auto_1</name>
        <target>soa_server1 (migratable)</target>
        <persistent-store>UMSJMSFileStore_auto_1</persistent-store>
      </jms-server>
      <jms-server>
        <name>SOAJMSServer_auto_1</name>
        <target>soa_server1 (migratable)</target>
        <persistent-store>SOAJMSFileStore_auto_1</persistent-store>
      </jms-server>
      <jms-server>
        <name>BPMJMSServer_auto_2</name>
        <target>soa_server2 (migratable)</target>
        <persistent-store>BPMJMSFileStore_auto_2</persistent-store>
        <message-buffer-size>100000</message-buffer-size>
      </jms-server>
      <jms-server>
        <name>UMSJMSServer_auto_2</name>
        <target>soa_server2 (migratable)</target>
        <persistent-store>UMSJMSFileStore_auto_2</persistent-store>
      </jms-server>
```
Persistent store should be set as per the model.

**Comment by CarolynRountree** [comment]
Could you please provide your osb/soa typedef file so I am using the same templates and schemas please.

**Comment by CarolynRountree** [comment]
fixed in 19.1.15
